spring/spring/pom.xml

```diff
@@ -237,6 +237,43 @@
                     </instructions>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
spring 4 works differently. XSD is in spring4-common, while tests are in spring4-embedded.
In such situation there's no problem with classpath resolution of the XSD.